debian/inadyn.conf

```diff
@@ -45,6 +45,13 @@
 #	hostname = <hostname>
 #}
 
+### FreeMyIP -- https://freemyip.com
+#provider default@freemyip.com {
+#        password = <your_token>
+#        hostname = <your_hostname>.freemyip.com
+#        ssl      = true
```

Comment:
`ssl = true` is the new black, no need to add it to new providers that support it.